src/AltinnCore/UnitTest/Common/JsonInstanceModelGenerationTest.cs

```diff
@@ -35,43 +37,86 @@ public void JsonInstanceModel()
 
             Assert.NotNull(instanceModel);
             JsonObject actualElements = instanceModel.TryGetObject("Elements");
-            Assert.Equal(158, actualElements.Count);
+            Assert.Equal(410, actualElements.Count);
 
             string metadataAsJson = instanceModel.ToString();
 
-            File.WriteAllText("edag1.instance-model.json", metadataAsJson);
+            //File.WriteAllText("edag1.instance-model.json", metadataAsJson);
         }
 
         /// <summary>
-        ///  nokså fantastisk
+        ///  fantastisk
         /// </summary>
         [Fact]
-        public void JsonInstanceFromAutogenJson()
+        public void TextGenerationOk()
         {
-            var schemaText = File.ReadAllText("Common/jsd/melding1.schema.json");
+            var schemaText = File.ReadAllText("Common/jsd/service-model.schema.json");
             var schemaJson = JsonValue.Parse(schemaText);
             var schema = new JsonSerializer().Deserialize<JsonSchema>(schemaJson);
 
-            JsonSchemaToInstanceModelGenerator converter = new JsonSchemaToInstanceModelGenerator("TestOrg", "edag", schema);
+            JsonSchemaToInstanceModelGenerator converter = new JsonSchemaToInstanceModelGenerator("TestOrg", "ServiceModel", schema);
 
-            JsonObject instanceModel = converter.GetInstanceModel();
+            ServiceMetadata serviceMetadata = converter.GetServiceMetadata();
 
-            Assert.NotNull(instanceModel);
-            JsonObject actualElements = instanceModel.TryGetObject("Elements");
-            Assert.Equal(101, actualElements.Count);
+            Assert.NotNull(serviceMetadata);
 
-            string metadataAsJson = instanceModel.ToString();
+            // get a chosen text identifier
+            string placeholder = serviceMetadata.Elements["klagefristgrp5804.KlageUtloptKlagefristBegrunnelsedatadef25456.value"].Texts["PlaceHolder"];
+            Assert.Equal("25456.KlageUtloptKlagefristBegrunnelsedatadef25456.PlaceHolder", placeholder);           
+        }
+        
+        /// <summary>
+        ///  nokså fantastisk
```

Comment:
Will there ever be meaningfull summary texts here? :sheep:

Reply:
Have fixed

Reply:
🎆